fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetadata.java

```diff
@@ -240,6 +246,52 @@ public List<RemoteFileInfo> getRemoteFileInfos(Table table, List<PartitionKey> p
         return fileOps.getRemoteFiles(partitions.build(), useRemoteFileCache);
     }
 
+    @Override
+    public List<RemoteFileInfo> getRemoteFileInfos(Table table, List<String> partitionNames) {
+        ImmutableList.Builder<Partition> partitions = ImmutableList.builder();
+        Map<String, Partition> existingPartitions = hmsOps.getPartitionByNames(table, partitionNames);
+        partitions.addAll(existingPartitions.values());
+        boolean useRemoteFileCache = true;
+        if (table instanceof HiveTable) {
+            useRemoteFileCache = ((HiveTable) table).isUseMetadataCache();
```

Comment:
do you know what this table could be? IcebergTable?

Reply:
yes, it could be HudiTable.